components/apimgt/org.wso2.carbon.apimgt.rest.api/src/gen/java/org/wso2/carbon/apimgt/rest/api/model/API.java

```diff
@@ -152,6 +153,30 @@ public void setResponseCaching(String responseCaching) {
   /**
    **/
   @ApiModelProperty(value = "")
+  @JsonProperty("cacheTimeout")
```

Comment:
Please note that these files get auto-generated using the swagger-to-cxf maven plugin we use for REST API development.